chemfiles-sys/lib.rs

```diff
@@ -164,17 +171,17 @@ extern "C" {
     pub fn chfl_topology_angles(topology: *const CHFL_TOPOLOGY, data: *mut [u64; 3], count: u64) -> chfl_status;
     pub fn chfl_topology_dihedrals(topology: *const CHFL_TOPOLOGY, data: *mut [u64; 4], count: u64) -> chfl_status;
     pub fn chfl_topology_impropers(topology: *const CHFL_TOPOLOGY, data: *mut [u64; 4], count: u64) -> chfl_status;
-    pub fn chfl_topology_clear_bonds(topology: *mut CHFL_TOPOLOGY) -> chfl_status;
     pub fn chfl_topology_add_bond(topology: *mut CHFL_TOPOLOGY, i: u64, j: u64) -> chfl_status;
     pub fn chfl_topology_remove_bond(topology: *mut CHFL_TOPOLOGY, i: u64, j: u64) -> chfl_status;
+    pub fn chfl_topology_clear_bonds(topology: *mut CHFL_TOPOLOGY) -> chfl_status;
     pub fn chfl_topology_residues_count(topology: *const CHFL_TOPOLOGY, count: *mut u64) -> chfl_status;
     pub fn chfl_topology_add_residue(topology: *mut CHFL_TOPOLOGY, residue: *const CHFL_RESIDUE) -> chfl_status;
     pub fn chfl_topology_residues_linked(topology: *const CHFL_TOPOLOGY, first: *const CHFL_RESIDUE, second: *const CHFL_RESIDUE, result: *mut c_bool) -> chfl_status;
     pub fn chfl_topology_bond_with_order(topology: *mut CHFL_TOPOLOGY, i: u64, j: u64, bond_order: chfl_bond_order) -> chfl_status;
     pub fn chfl_topology_bond_orders(topology: *const CHFL_TOPOLOGY, orders: *mut chfl_bond_order, nbonds: u64) -> chfl_status;
     pub fn chfl_topology_bond_order(topology: *const CHFL_TOPOLOGY, i: u64, j: u64, order: *mut chfl_bond_order) -> chfl_status;
     pub fn chfl_cell(lengths: *const c_double, angles: *const c_double) -> *mut CHFL_CELL;
-    pub fn chfl_cell_from_matrix(matrix: *const [c_double; 3]) -> *mut CHFL_CELL;
+    pub fn chfl_cell_from_matrix(matrix: *mut [c_double; 3]) -> *mut CHFL_CELL;
```

Comment:
Good point for using bindgen. At the time of updating I thought that this must be const because the array in the C API is const, but turns out I [overlooked the pointer weirdness of C](https://cppinsights.io/lnk?code=dm9pZCBmb28oY29uc3QgaW50IHBbM10pIHsKICBwID0gbnVsbHB0cjsKICAvLyBwWzFdID0gMTA7IC8vIE5vdCBhbGxvd2VkCn0KCnZvaWQgZm9vMihjb25zdCBpbnQqIHApIHsKICBwID0gbnVsbHB0cjsKICAvLyBwWzFdID0gMTA7IC8vIE5vdCBhbGxvd2VkCn0KCnZvaWQgZm9vMyhjb25zdCBpbnQgKmNvbnN0IHApIHsKICAvLyBwID0gbnVsbHB0cjsgLy8gTm90IGFsbG93ZWQKICAvLyBwWzFdID0gMTA7IC8vIE5vdCBhbGxvd2VkCn0KCnZvaWQgZm9vNChjb25zdCBpbnQgKCZwKVszXSkgewogIC8vIHAgPSBudWxscHRyOyAvLyBOb3QgYWxsb3dlZAogIC8vIHBbMV0gPSAxMDsgLy8gTm90IGFsbG93ZWQKfQoKaW50IG1haW4oKQp7CiAgICBpbnQgeFszXSA9IHsxLCAyLCAzfTsKICAJZm9vKHgpOwogIAlmb28yKHgpOwogIAlmb28zKHgpOwogIAlmb280KHgpOwp9&insightsOptions=cpp17&std=cpp17&rev=1.0) :D